Alignment/CommonAlignmentProducer/python/ALCARECOMuAlCalIsolatedMu_Output_cff.py

```diff
@@ -7,6 +7,7 @@
     ),
     outputCommands = cms.untracked.vstring(
         'keep *_ALCARECOMuAlCalIsolatedMu_*_*', 
+        'keep *_generalTracks_*_*',
```

Comment:
@pakhotin @anthonykrose any update on the question above and the estimate of the events size?
Thanks